core/model/src/test/java/org/eclipse/sw360/antenna/model/test/ClassCodeSourceLocationTest.java

```diff
@@ -0,0 +1,30 @@
+package org.eclipse.sw360.antenna.model.test;
+
+import org.eclipse.sw360.antenna.model.util.ClassCodeSourceLocation;
+import org.junit.Test;
+
+import java.net.URISyntaxException;
+import java.nio.file.Paths;
+
+import static junit.framework.TestCase.fail;
+
+public class ClassCodeSourceLocationTest {
+
+    @Test
+    public void pathFromStringDoesntThrowInvalidPathExceptionTest() {
+        try {
+            Paths.get(ClassCodeSourceLocation.getClassCodeSourceLocationAsString(this.getClass()));
```

Comment:
I would expect some assertions on the result (which help me to understand what the function does). E.g. something like:
- the path should exist
- the path should end with
- ...